woocommerce/woocommerce
# [testing workflows] Move PHP tests with WP nighly back into ci.yml

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).

### Changes proposed in this Pull Request:

With [optional tests now supported](https://github.com/woocommerce/woocommerce/issues/45097) in `ci.yml` we can move the WP nightly back. They were removed by [#45096](https://github.com/woocommerce/woocommerce/pull/45096) and then added to a cron workflow with [#45260](https://github.com/woocommerce/woocommerce/pull/45260). 

### How to test the changes in this Pull Request:

Check CI, make sure `PHP WP: nightly` are running for `@woocommerce/plugin-woocommerce`. The job should be optional.
